src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -59,6 +59,9 @@ def acr_create(cmd,
     admin_user_enabled = admin_enabled == 'true'
 
     if sku == SkuName.classic.value:
+        logger.warning(
+            "Due to the planned deprecation of the Classic registry SKU, we recommend using "
+            "Basic, Standard, or Premium for all new registries. See aka.ms/acr/skus for details.")
```

Comment:
Add `https://` to URL for convenience? Some shells support CTRL+click to open in browser window.